src/coreComponents/events/EventManager.cpp

```diff
@@ -75,6 +76,10 @@ EventManager::EventManager( string const & name,
     setRestartFlags( RestartFlags::WRITE_AND_READ ).
     setDescription( "Index of the current subevent." );
 
+  registerWrapper( viewKeyStruct::timeOutputFormat(), &m_timeOutputFormat ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setRestartFlags( RestartFlags::NO_WRITE ).
```

Comment:
Despite the `NO_WRITE` flag, check if this still needs a rebaseline. IIRC, each registered wrapper still creates a Conduit node and an HDF5 group in the restart files, regardless of whether data is actually written.

Reply:
your are right, unfortunately a rebaseline is necessary.